package.json

```diff
@@ -6,6 +6,10 @@
     "type": "git",
     "url": "https://github.com/berryboy/chrono.git"
   },
+  "jest": {
```

Comment:
This was to fix [this error](https://stackoverflow.com/questions/51554366/jest-securityerror-localstorage-is-not-available-for-opaque-origins#answer-51554619) which was causing all the unit tests to fail.

I figure it'd be good to add to the project to make it a bit easier in case others run into this.